lib/board.rb

```diff
@@ -0,0 +1,22 @@
+class Board
+  attr_reader :new_array
+
+  def initialize
+    @new_array = ['-', '-', '-', '-', '-', '-', '-', '-', '-']
+  end
+
+  def display_board(array_my = nil)
+    display_array = array_my.nil? ? @new_array : array_my
+    count = 0
+    display_array.each do |x|
+      print x
+      count += 1
+      if count == 3
+        puts
+        count = 0
+      else
+        print ' | '
+      end
+    end
+  end
```

Comment:
* [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- instead of using puts and print, you could have a String variable lets call it `result` for example to concatenate everything that you are printing and at the end of your method retrieve it and call it in your main.rb file like this:
```ruby
puts display_board()
```